rest-api/api-docs/bridges.json

```diff
@@ -51,6 +51,15 @@
 							"required": false,
 							"allowMultiple": false,
 							"dataType": "string"
+						},
+						{
+							"name": "sendSDPLabel",
```

Comment:
I don't think this should be a field on the bridge create itself. I think it should be another property on the type. These define the characteristics of the bridge, and sending stream labels is a characteristic of it.

Reply:
Okay, I'll have a look into that and see if I can rework it.